README.md

```diff
@@ -6,23 +6,13 @@
 
 Fluentd plugin to extract key/values from URL query parameters.
 
-## Synopsis
-
-Imagin you have a config as below:
+## Caveat
 
-fluentd `< 0.12`:
-
-```
-<match test.**>
-  @type extract_query_params
+This plugin, from version 0.1.0, supports fluentd of version 0.14 or higher.
```

Comment:
Could you add additional line for v0.12 like below?
`If you use fluentd v0.12 or earlier, use v0.0.x versions`